package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@yext/search-core",
-  "version": "1.8.1",
+  "version": "1.8.2",
```

Comment:
you should bump the version to a beta, like 1.9.0-#whateverPrNumberThisIs (I'm on github mobile right now)

This would be 1.9.0 not 1.8.2 because this is a feature not a bug fix

I was thinking you would bump it in a separate PR, but this also works!

Reply:
oke! bumped it back to 1.8.1 and we can bump version in a separate PR